sys-apps/ignition/files/0006-config-v3_4-convert-ignition-2.x-to-3.4.patch

```diff
@@ -1,24 +1,24 @@
-From ab4a9ade5b2e43a4d164392becf8d421256d8ffa Mon Sep 17 00:00:00 2001
+From 352d5bbd1062d8affa730ae88aff8cd9d240f44f Mon Sep 17 00:00:00 2001
 From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
 Date: Wed, 25 May 2022 10:20:09 +0200
-Subject: [PATCH 06/16] config/v3_4: convert ignition 2.x to 3.4-exp
+Subject: [PATCH 06/18] config/v3_4: convert ignition 2.x to 3.4
 
-if the version of ignition is 2.x we convert it to 3.4-exp using
+if the version of ignition is 2.x we convert it to 3.4 using
 ign-converter.
 
 it should support any 2.x version (or at the least the last 2 releases)
 
 Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
 ---
- config/v3_4_experimental/config.go | 37 ++++++++++++++++++++++++++++++
+ config/v3_4/config.go | 37 +++++++++++++++++++++++++++++++++++++
  1 file changed, 37 insertions(+)
 
-diff --git a/config/v3_4_experimental/config.go b/config/v3_4_experimental/config.go
-index 097dbd83..e666c876 100644
---- a/config/v3_4_experimental/config.go
-+++ b/config/v3_4_experimental/config.go
+diff --git a/config/v3_4/config.go b/config/v3_4/config.go
+index 0a01a560..32770331 100644
+--- a/config/v3_4/config.go
++++ b/config/v3_4/config.go
```

Comment:
Why is this 3.4, I think it should be 3_5_experimental because that's the version parsed by `config/config.go`

Reply:
(If that is the case I wonder how the kola tests for spec 1 and 2 passed)

Reply:
Having the translation step somewhere in the middle of the `ParseCompatibleVersion` chain is a bit confusing even if it now works. I suggest to either do it at the top as before or try to move it to `config/v3_0/config.go` which would be nice but is more work because the translator doesn't use 3.0 as far as I remember.

Reply:
Thanks!